Formula/b/bower-email.rb

```diff
@@ -0,0 +1,35 @@
+class BowerEmail < Formula
+  desc "Curses terminal client for the Notmuch email system"
+  homepage "https://github.com/wangp/bower"
+  url "https://github.com/wangp/bower/archive/refs/tags/1.1.1.tar.gz"
+  sha256 "4c041681332d355710aa2f2a935ea56fbb2ba8d614be81dee594c431a1d493d9"
+  license "GPL-3.0-only"
+
+  depends_on "mercury" => :build
+  depends_on "pandoc" => :build
+
+  depends_on "coreutils"
+  depends_on "gpgme"
+  depends_on "ncurses"
+  depends_on "notmuch"
+
+  def install
+    system "make"
+    system "make", "man"
+    bin.install "bower" => "bower-email"
+    man1.install "bower.1" => "bower-email.1"
+  end
+
+  def caveats
+    <<~EOS
+      This formula installs the binary as "bower-email" to avoid conflicts with the "bower" package.
+      If you prefer to use it as "bower", you can create a new symlink:
+        brew unlink bower-email
+        ln -s #{opt_bin}/bower-email #{HOMEBREW_PREFIX}/bin/bower
+    EOS
+  end
```

Comment:
We can just add a `conflicts_with` statement instead. We can't do it in this PR, because it would require the other `bower` formula to be rebuild. We can follow up with a syntax only PR to introduce it.